prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -126,17 +132,21 @@ public void onChange(Entity oldEntity, Entity newEntity) throws FalconException{
                 backlogMetricStore.deleteEntityInstance(newProcess.getName());
                 entityBacklogs.remove(newProcess);
             }
+            Process process = EntityUtil.getEntity(oldEntity.getEntityType(), oldEntity.getName());
```

Comment:
Understand that we should stop emitting metrics for deleted process. Why should we reset to 0? Basically, if I look back at the graph, it should tell me that at the point of delete the backlog was so much.